README.md

```diff
@@ -48,6 +48,16 @@ Documentation
 --------------
 Documentation, including quick start, architecture, use cases, API docs, and much more, is available on [ReadTheDocs](https://opentimelineio.readthedocs.io/)
 
+Support Platforms
```

Comment:
Should this be `Supported Platforms`? Support platforms almost looks like it's about where can users get support (slack, GitHub, etc).

Reply:
how about `Supported VFX Platforms` instead?  Since its really just talking about vfx platform.

Reply:
Even better!

Reply:
```suggestion
Supported VFX Platforms
```